nodejs/week3/meal_sharing/src/backend/api/meals.js

```diff
@@ -0,0 +1,189 @@
+const { request, response, query } = require("express");
+const { sum, limit, select, groupBy } = require("../database");
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+// Getting all meals
+router.get("/", async(request, response) => {
```

Comment:
Combine this router with the one with the filters (that has maxPrice, title, etc.) starting on line 150.